pkgs/development/libraries/libgphoto2/default.nix

```diff
@@ -33,8 +48,6 @@ stdenv.mkDerivation rec {
       MTP, and other vendor specific protocols for controlling and transferring data
       from digital cameras.
     '';
-    version = "2.5.23";
-    tag = "libgphoto2-2_5_23-release";
     # XXX: the homepage claims LGPL, but several src files are lgpl21Plus
```

Comment:
What does the XXX: mean?

Reply:
It's [a way to indicate a sort of TODO or ugliness note](https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments), but I don't think it's good, exactly because it isn't obvious what it means.

Reply:
I recognized the writing style as mine, but I didn't remember writing that comment. 

Git blame claims I wrote it 8 years ago!

The upstream project now reports: [GNU Lesser General Public License v2.1](https://github.com/gphoto/libgphoto2/blob/master/COPYING). Let's use that.

Reply:
@jcumming Unfortunately, the GitHub is not able to detect if the license is `*gpl-or-later` or `*gpl-only`, see https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
I'll leave it to you people to figure that one out :grin: 

Beyond that, anybody mind approving if it looks good?